mlrun/common/schemas/model_monitoring/constants.py

```diff
@@ -289,6 +289,11 @@ class ModelMonitoringMode(StrEnum):
     disabled = "disabled"
 
 
+class ScheduleChiefFields(MonitoringStrEnum):
```

Comment:
If you don't use `.list()`, the following is clearer:
```suggestion
class ScheduleChiefFields(StrEnum):
```